samples/apps/copilot-chat-app/webapi/Config/DocumentMemoryOptions.cs

```diff
@@ -46,4 +46,10 @@ public class DocumentMemoryOptions
     /// </summary>
     [Range(0, int.MaxValue)]
     public int FileSizeLimit { get; set; } = 1000000;
+
+    /// <summary>
+    /// Similarity threshold to avoid document memory duplication when importing documents to memory.
+    /// </summary>
+    [Range(0.0, 1.0)]
+    public double DeduplicationSimilarityThreshold { get; set; } = 0.8;
```

Comment:
Could you add a comment on the logic of the default value? I naively feel its low for dedupe.

Reply:
It's actually the opposite. The lower the value, the stronger the deduplication will be.
I have added a comments in both the source and the setting Json.